.eslintrc.json

```diff
@@ -8,6 +8,7 @@
 	},
 	"plugins": ["unicorn"],
 	"root": true,
+	"ignorePatterns": ["shared/"],
```

Comment:
what's the purpose of this line? if it's specific to `create-svelte/shared`, should we make that explicit?

Reply:
ok. I put the full path

without it I get:
```
ESLint couldn't find the config "prettier" to extend from. Please check that the name of the config is correct.

The config "prettier" was referenced from the config file in "packages/create-svelte/shared/+eslint+prettier+typescript/.eslintrc.cjs".
```